pulumi/tools/chocolateyinstall.ps1

```diff
@@ -8,4 +8,17 @@ $packageArgs = @{
   file64         = "$toolsDir\pulumi-vX.X.X-windows-x64.zip"
 }
 
-Install-ChocolateyZipPackage  @packageArgs
\ No newline at end of file
+Install-ChocolateyZipPackage  @packageArgs
+
+$bindir = Join-Path -Path $toolsDir -ChildPath "bin"
+
+# create empty sidecar so shimgen ignores them.
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-python.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-nodejs.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-go.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-dotnet.exe.ignore") -Value $null
```

Comment:
```suggestion
Get-ChildItem $bindir\*.exe | ForEach-Object { Set-Content -Value "" -Path "$_.ignore" }
```

This could be done in one line. When a new exe file is added to the archive, no changes to the package are required.